README.md

```diff
@@ -85,4 +85,13 @@ rules-bot will automatically delete the service messages announcing new members.
 
 ## Other
 
-rules-bot can make sandwiches. You can ask it to do so by typing `make me a sandwich`. We'll see, if it actually does 😉
\ No newline at end of file
+rules-bot can make sandwiches. You can ask it to do so by typing `make me a sandwich`. We'll see, if it actually does 😉
+
+## Setting up the bot for development and testing
```

Comment:
```suggestion
# Setting up the bot for development and testing
```

Reply:
Ok, making both changes (this one and next one) in a separate commit